SPECS-EXTENDED/suitesparse/suitesparse.spec

```diff
@@ -0,0 +1,752 @@
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+%global amd_version_major 2
+%global btf_version_major 1
+%global camd_version_major 2
+%global ccolamd_version_major 2
+%global cholmod_version_major 3
+%global colamd_version_major 2
+%global csparse_version_major 3
+%global cxsparse_version_major 3
+%global klu_version_major 1
+%global ldl_version_major 2
+%global rbio_version_major 2
+%global spqr_version_major 2
+%global SuiteSparse_config_major 5
+%global umfpack_version_major 5
+
+### CXSparse is a superset of CSparse, and the two share common header
+### names, so it does not make sense to build both. CXSparse is built
+### by default, but CSparse can be built instead by defining
+### enable_csparse as 1 below.
+%global enable_csparse 0
+
+# Whether to build a separate version of libraries linked against an ILP64 BLAS
+%if 0%{?__isa_bits} == 64
+%global build64 1
+%endif
```

Comment:
Since we're not building for architectures, where `0%{?__isa_bits} == 64` would be false, let's simplify the spec by removing these checks and leaving only the parts for `%{build64} == 1`.